simphony/engine/__init__.py

```diff
@@ -3,9 +3,62 @@
 This module is dynamicaly populated at import with the
 registered plugins modules. Plugins modules need to be
 registered at the 'simphony.engine' entry point.
+"""
+from .extension import ABCEngineExtension
+from .extension import EngineInterface
+from .extension import EngineManager
 
 
-"""
+__all__ = ['ABCEngineExtension', 'EngineInterface',
+           'get_supported_engines', 'create_wrapper',
+           'get_supported_engine_names']
+
+
+# TODO: Use an application server and put this in app context.
+# Wrapper manager class.
+_ENGINE_MANAGER = EngineManager()
+
+
+def get_supported_engine_names():
+    """Show a list of supported engine names.
+
+    Returns
+    -------
+    names: list
+        a list of engine names.
+    """
+    return _ENGINE_MANAGER.get_supported_engine_names()
+
+
+def get_supported_engines():
+    """Show a list of supported engines.
+
+    Returns
+    -------
+    metadata: list
+        a list of engine metadata objects
+    """
+    return _ENGINE_MANAGER.get_supported_engines()
+
+
+def create_wrapper(cuds, engine_name, engine_interface=None):
+    """Create a wrapper to the given engine.
+
+    Parameters
+    ----------
+    cuds: CUDS
+        A cuds object which contains model information.
```

Comment:
I'd suggest remove capital first letter and full stop at the end from these lines. They are not really proper sentences that need those, and it would be consistent with earlier docstrings.